components/butterfly/src/server/pull.rs

```diff
@@ -16,17 +16,32 @@
 //!
 //! This module handles pulling all the pushed rumors from every member off a ZMQ socket.
 
-use std::sync::atomic::Ordering;
 use std::thread;
 use std::time::Duration;
 
+use prometheus::{CounterVec, GaugeVec};
 use zmq;
 
 use rumor::{RumorEnvelope, RumorKind};
 use server::Server;
 use trace::TraceKind;
 use ZMQ_CONTEXT;
 
+lazy_static! {
+    static ref GOSSIP_MESSAGES_RECEIVED: CounterVec = register_counter_vec!(
+        "hab_butterfly_gossip_messages_received_total",
```

Comment:
Like previous comment, consider renaming to _count or removing _total since this is a counter